### ISSUE TITLE
Does not work with Thunderbird 91.8.1

### ISSUE THREAD

**Issue by daspraanjal** [spymastermatt/thunderbird-monterail#127]
Did everything correctly but it is not working, nothing changes.

**Comment by eli-xciv** [comment]
Same with `102.11.0`

**Comment by JohnPBee** [comment]
Guess it goes without saying, this theme is not longer applicable - too bad, it looked so good

**Comment by qwertychouskie** [comment]
Agreed, nowadays I use https://github.com/rafaelmardojai/thunderbird-gnome-theme, but modified to use the icons from this theme.

[chrome.zip](https://github.com/spymastermatt/thunderbird-monterail/files/14130740/chrome.zip)
